dbt_platform_helper/utils/versioning.py

```diff
@@ -97,19 +111,31 @@ def get_platform_helper_versions() -> PlatformHelperVersions:
     parsed_released_versions = [parse_version(v) for v in released_versions]
     parsed_released_versions.sort(reverse=True)
     latest_release = parsed_released_versions[0]
+    platform_config = load_and_validate_platform_config(disable_aws_validation=True)
+    platform_config_default = parse_version(
+        platform_config.get("default_versions", {}).get("platform-helper")
+    )
+
+    pipeline_overrides = {
+        name: pipeline.get("versions", {}).get("platform-helper")
+        for name, pipeline in platform_config.get("environment_pipelines", {}).items()
+        if pipeline.get("versions", {}).get("platform-helper")
+    }
 
     version_from_file = None
-    message = "Cannot get dbt-platform-helper version from file '.platform-helper-version'. Check if file exists."
+    message = f"Cannot get dbt-platform-helper version from file '{PLATFORM_HELPER_VERSION_FILE}'. Check if file exists."
 
     try:
-        version_from_file = parse_version(Path(".platform-helper-version").read_text())
+        version_from_file = parse_version(Path(PLATFORM_HELPER_VERSION_FILE).read_text())
     except FileNotFoundError:
         click.secho(f"{message}", fg="yellow")
```

Comment:
For readability, could move the message into the except block since its only relevant to the exception case.  Also f string is currently not needed so could be removed or the message could be in-lined:
```
    try:
        version_from_file = parse_version(Path(PLATFORM_HELPER_VERSION_FILE).read_text())
    except FileNotFoundError:
        message = f"Cannot get dbt-platform-helper version from file '{PLATFORM_HELPER_VERSION_FILE}'. Check if file exists."
        click.secho(message, fg="yellow")
```

Reply:
Agreed. I just introduced the constant. However I have completely refactored the messaging here in the last commit